scss/partials/_header.scss

```diff
@@ -0,0 +1,363 @@
+/* ---------------------------------------------------- */
+/* ----- HEADER ----- */
+/* ---------------------------------------------------- */
+
+.slider {
+  background-size: cover;
+  position: relative;
+
+  &__slide {
+    position: relative;
+
+    &__title {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      text-align: center;
+      transform: translateY(60px);
+      font-size: 2.125rem;
+      font-weight: 900;
+
+      @media #{$tablet__768} {
```

Comment:
Zróv to w ten sposób `@media (min-width: 950px) {
        padding: 50px 0;
    }`